docs/pages/integrating/vc.rst

```diff
@@ -118,9 +118,12 @@ The following query does not return all `NutsOrganizationCredential` but **all**
     }
 
 To find certain credentials, you'll need to add fields that are required to exist in the desired credential.
-The following query searches for credentials that have a organization name that starts with an empty string.
-Any credential that does not have an organization name will be ignored.
-By default, field selection is done by matching the given value as prefix.
+By default, matching is exact: it only returns the result when the given value exactly matches.
+There are 2 other matchers for strings:
+- ``"*"`` to match credential fields that contain the field (non-empty)
+- ``*`` as postfix to match credential fields that start with the given string, e.g. ``Hospital Amst*``
```

Comment:
Also add that prefix wildcards are not supported, only postfix

Reply:
Also numbers and booleans don't support it. They are always exact match

Reply:
I'll make it explicit.